plugin/www/AppboyPlugin.js

```diff
@@ -93,6 +93,14 @@ AppboyPlugin.prototype.submitFeedback = function (email, feedback, isBug) {
 
 // Appboy user methods
 /**
+ * Sets the user Attribution Data. We are using this to connect our integration with Adjust
```

Comment:
Can we modify this to remove the second sentence so that the method remains generic for attribution data.